ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKSourceFactoryTest.java

```diff
@@ -64,6 +69,8 @@
       .build();
 
   private static final String TOPIC_NAME = "fred";
+  private static final int OLD_PSEUDOCOLUMN_VERSION = SystemColumns.ROWTIME_PSEUDOCOLUMN_VERSION;
```

Comment:
Some of the tests added in this file fail unless `OLD_PSEUDOCOLUMN_VERSION` is exactly `SystemColumns.LEGACY_PSEUDOCOLUMN_VERSION_NUMBER`. In other words, it doesn't work for this test file to use a generic "old" version and a generic "new version". As such, I think the tests should just use `LEGACY_PSEUDOCOLUMN_VERSION_NUMBER` directly, instead of defining an alias. Similarly, we can just use `SystemColumns.CURRENT_PSEUDOCOLUMN_VERSION_NUMBER` in place of defining a new variable `NEW_PSEUDOCOLUMN_VERSION` as well.